connection/local_executor.py

```diff
@@ -14,12 +14,15 @@
 
 class LocalExecutor(BaseExecutor):
     def _execute(self, command, timeout):
-        bash_path = TestRun.config.get("bash_path", "/bin/bash")
+        executable_path = TestRun.config.get(
+            "executable_path",
+            "/bin/bash" if os.name == 'posix' else None
+        )
```

Comment:
For sake of readability, how about
```
default_executable_path = "/bin/bash" if os.name == 'posix' else None
executable_path = TestRun.config.get("executable_path", default_executable_path)
```

Reply:
BTW the variable `bash_path` is also used in `_rsync()` method. Perhaps we should consider resolving the path in constructor?

Reply:
Also, `executable_path` -> `cli_path`

Reply:
I suggest that after https://github.com/Open-CAS/test-framework/pull/6 is merged we should remove rsync and replace it with copy* implemented there

Reply:
https://github.com/Open-CAS/test-framework/pull/6 is merged now, so this could be updated as well :)